dev/devicelab/manifest.yaml

```diff
@@ -76,13 +76,12 @@ tasks:
     stage: devicelab_win
     required_agent_capabilities: ["windows/android"]
 
-  # TODO(jonahwilliams): re-enable with https://github.com/flutter/flutter/issues/39597
-  # codegen_integration_win:
-  #   description: >
-  #     Runs codegeneration and verifies that it can execute
-  #     correctly.
-  #   stage: devicelab_win
-  #   required_agent_capabilities: ["windows/android"]
+   codegen_integration_win:
```

Comment:
i think this is an indentation error? From Cirrus:

```
  Error on line 79, column 4: Expected a key while parsing a block mapping.
     ,
  79 |    codegen_integration_win:
     |    ^
     '
```

https://cirrus-ci.com/task/6173665142243328?command=main#L141

Reply:
(yaml is hard)